### PR TITLE
Add dlp v2beta1 configs

### PR DESCRIPTION
**Pull request by eoogbe**

### CODE REVIEW

File: gapic/api/artman_dlp.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  src_proto_path:
+    - ${GOOGLEAPIS}/google/privacy/dlp/v2beta1
+  service_yaml:
+    - ${GOOGLEAPIS}/google/privacy/dlp/dlp.yaml
+  gapic_api_yaml:
+    - ${GOOGLEAPIS}/google/privacy/dlp/v2beta1/dlp_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
```
Comment (lukesneeringer) [inline gapic/api/artman_dlp.yaml]: Are we developing this in the open, or privately?

**Comment by garrettjonesgoogle** [comment]
If you want a second reviewer, have @michaelbausor take a look, I won't get a chance to.

File: google/privacy/dlp/v2beta1/dlp_gapic.yaml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  collections:
+  - name_pattern: inspect/results/{result}
+    entity_name: result
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
```
Comment (michaelbausor) [inline google/privacy/dlp/v2beta1/dlp_gapic.yaml]: Should this contain `UNAVAILABLE`?

File: google/privacy/dlp/v2beta1/dlp_gapic.yaml [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      groups:
+      - parameters:
+        - language_code
+    required_fields:
+    - language_code
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+- name: google.longrunning.Operations
```
Comment (michaelbausor) [inline google/privacy/dlp/v2beta1/dlp_gapic.yaml]: We don't normally have an interface config for long running operations - see e.g. Speech. Do we need one in this case?

**Pull request merged**